tests/sentry/sentry_apps/services/test_app_request.py

```diff
@@ -0,0 +1,91 @@
+from sentry.sentry_apps.services.app_request import SentryAppRequestFilterArgs, app_request_service
+from sentry.testutils.cases import TestCase
+from sentry.testutils.factories import Factories
+from sentry.testutils.pytest.fixtures import django_db_all
+from sentry.testutils.silo import control_silo_test, create_test_regions
+from sentry.utils.sentry_apps import SentryAppWebhookRequestsBuffer
+
+
+@django_db_all(transaction=True)
+@control_silo_test(regions=create_test_regions("us"))
+class TestRegionApp(TestCase):
+    def setUp(self):
+        self.user = Factories.create_user()
+        self.org = Factories.create_organization(owner=self.user, region="us")
+        self.app = Factories.create_sentry_app(
+            name="demo-app",
+            user=self.user,
+            organization=self.org,
+            published=True,
+            schema={
+                "elements": [
+                    {
+                        "type": "alert-rule-trigger",
+                        "title": "go beep",
+                        "settings": {
+                            "type": "alert-rule-settings",
+                            "uri": "https://example.com/search/",
+                        },
+                    },
+                ]
+            },
+        )
+
+    def test_get_buffer_requests_for_region(self):
```

Comment:
Nice job with testing! It may be good to have a test that also validates an empty buffer requests list.